opm/autodiff/FlowMain.hpp

```diff
@@ -101,6 +101,33 @@
 namespace Opm
 {
 
+    boost::filesystem::path simulationCaseName( const std::string& casename ) {
+        namespace fs = boost::filesystem;
+
+        const auto exists = []( const fs::path& f ) {
```

Comment:
>> This assumes C++14.
>
> Weird, because it compiled without question

Probably because your compilers does not actually implement the strict C++11 rules, but rather the later update that includes the resolution of a [defect report](http://stackoverflow.com/questions/28955478/when-can-we-omit-the-return-type-in-a-c11-lambda).  To be safe we should still specify the explicit return type in this case (unless we want to assume C++14 or that all compilers we'll ever encounter have implemented the resolved defect report).